### PR TITLE
fix(layout): night mode background color now fits full screen

### LINKED ISSUE #203: UI: Whitespace below map on dark mode
There's a bunch of whitespace below the map on dark mode when all categories are collapsed 
screen shot 2018-06-29 at 9 45 43 am
perhaps set a min-height of 100vh

Firefox 60.0.1

### PR DESCRIPTION
**Pull request by gforce2k5**
adding 100vh property to layout.css in order to make the background color fill the browser window
Closes #203

### CODE REVIEW

File: src/layouts/layout.css [hunk 1]
```diff
@@ -28,4 +28,5 @@ main {
 
 .app-wrapper {
   margin-top: 38px;
+  height: 100vh;
```
Comment (Bouncey) [inline src/layouts/layout.css]: Can we change this to 

```css
.app-wrapper {
  margin-top: 38px;
  height: calc(100vh - 38px);
}
```

Else we get some white space under the challenge views

**Comment by gforce2k5** [comment]
i'll fix it

**Comment by gforce2k5** [comment]
@Bouncey I changed what you asked for

**Comment by Bouncey** [comment]
Thanks @gforce2k5, this fixes the challenge view, but now if the map is expanded and I scroll down, I get this 

image

**Comment by gforce2k5** [comment]
@Bouncey found the problem and fixed, tested locally and it works good (changed height to min-height)

**Pull request merged**